packages/generator/src/edmx-parser/common/edmx-parser.ts

```diff
@@ -27,9 +27,36 @@ export function parseEntityTypesBase(root: any): EdmxEntityTypeBase<any>[] {
     },
     NavigationProperty: forceArray(e.NavigationProperty),
     Property: forceArray(e.Property),
-    Namespace: e.Namespace
+    Namespace: e.Namespace,
+    BoundFunction: extractBoundFunctions(root),
+    BoundAction: extractBoundActions(root)
   }));
 }
+
+function extractBoundFunctions(root: any) {
+  const x = root[0];
```

Comment:
q: how many items does this list have? What does that mean semantically? is there a proper way to access the "correct" object?